src/library/list.lisp

```diff
@@ -415,6 +415,11 @@
            (any f xs)))
       ((Nil) False)))
 
+  (declare split (Char -> String -> (List String)))
+  (define (split c str)
+    (lisp (List String) (c str)
+      (uiop:split-string str :separator `(,c))))
```

Comment:
Add `#:UIOP` to .asd as a dep.

Use `(list c)` instead of quasiquoting, or better yet

```
(let ((split-chars (list c)))
  (declare (dynamic-extent split-chars))
  (uiop:...
```